dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/domain/TrackedEntity.java

```diff
@@ -45,8 +45,10 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class TrackedEntity
+public class TrackedEntity implements TrackerDto
 {
+    private String uid;
```

Comment:
UID should be the same as trackedEntity

Reply:
This uid is used as a temp holder for the new uid when separating between create/update